api/src/test/java/org/openmrs/validator/PersonValidatorTest.java

```diff
@@ -166,8 +185,7 @@ public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() throws Ex
 		person.setBirthdate(new Date());
 		person.setGender("too long text too long too long text too long text  too long text");
 		person.setPersonVoided(true);
-		person
```

Comment:
We only recommend doing mvn clean install on the whole project. This should not have any formatting in places you have not touched.